File: DLTools/src/main/java/com/toune/dltools/dialog/MediaLoader.java
package com.toune.dltools.dialog;

import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.bumptech.glide.request.RequestOptions;
import com.toune.dltools.R;
import com.yanzhenjie.album.AlbumFile;
import com.yanzhenjie.album.AlbumLoader;

public class MediaLoader implements AlbumLoader {

    @Override
    public void load(ImageView imageView, AlbumFile albumFile) {
        load(imageView, albumFile.getPath());
    }

    @Override
    public void load(ImageView imageView, String url) {
        Glide.with(imageView.getContext())
                .load(url)
                .apply(new RequestOptions()
                        .error(R.drawable.logo)
                        .placeholder(R.drawable.logo)
                )
                .into(imageView);
    }
}

File: DLTools/src/main/java/com/toune/dltools/dialog/MediaLoader.kt
package com.toune.dltools.dialog

import android.widget.ImageView
import com.bumptech.glide.Glide
import com.bumptech.glide.request.RequestOptions
import com.toune.dltools.R
import com.yanzhenjie.album.AlbumFile
import com.yanzhenjie.album.AlbumLoader

class MediaLoader : AlbumLoader {
    override fun load(imageView: ImageView, albumFile: AlbumFile) {
        load(imageView, albumFile.path)
    }

    override fun load(imageView: ImageView, url: String) {
        Glide.with(imageView.context)
            .load(url)
            .apply(
                RequestOptions()
                    .error(R.drawable.logo)
                    .placeholder(R.drawable.logo)
            )
            .into(imageView)
    }
}